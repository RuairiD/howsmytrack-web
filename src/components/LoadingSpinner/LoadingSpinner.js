import React from 'react';
import { Spin } from 'antd';
import { Div, Img } from 'lemon-reset';

const LoadingSpinner = () => (
    <Div className="home-loading-container">
        <Spin 
            className="home-loading-spin"
            size="large"
            indicator={(
                <Img
                    alt=""
                    src="/logo128.png"
                />
            )}
        />
    </Div>
);

export default LoadingSpinner;