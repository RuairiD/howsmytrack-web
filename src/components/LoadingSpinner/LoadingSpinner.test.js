import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import LoadingSpinner from "./LoadingSpinner";

describe("LoadingSpinner", () => {
    it("renders a Spin", () => {
        const wrapper = shallow(
            <LoadingSpinner />,
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
