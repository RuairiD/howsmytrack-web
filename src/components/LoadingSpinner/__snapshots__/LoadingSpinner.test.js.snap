// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner renders a Spin 1`] = `
<Div
  className="home-loading-container"
>
  <Spin
    className="home-loading-spin"
    indicator={
      <Img
        alt=""
        className=""
        src="/logo128.png"
      />
    }
    size="large"
    spinning={true}
    wrapperClassName=""
  />
</Div>
`;
